signals.mml

```diff
@@ -387,7 +387,8 @@ Layer:
             tags->'railway:signal:crossing:distant:type' AS crossing_distant_type,
             tags->'railway:signal:crossing:type' AS crossing_type,
             tags->'railway:signal:ring:type' AS ring_type,
-            tags->'railway:signal:whistle:type' AS whistle_type
+            tags->'railway:signal:whistle:type' AS whistle_type,
+            tags->'railway:signal:train_protection:shape' AS shape
```

Comment:
This needs to be ```train_protection_shape```, otherwise it will collide with any other shape that may be used.

Reply:
You misread this. The line must be:

```tags->'railway:signal:train_protection:shape' AS train_protection_shape```